<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="BaseGLWidget" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Object</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Sphere</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cone</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cylinder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ObjFile</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="colorButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Change Color</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolBox" name="tools">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>141</width>
         <height>231</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>123</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Select</string>
        </attribute>
        <widget class="QListWidget" name="objectList">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>141</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>123</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Translation</string>
        </attribute>
        <widget class="QSlider" name="Tx">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Ty">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Tz">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>123</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Rotation</string>
        </attribute>
        <widget class="QSlider" name="Rz">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Rx">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Ry">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>123</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Scale</string>
        </attribute>
        <widget class="QSlider" name="Sz">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Sx">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Sy">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>140</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Selected</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseGLWidget</class>
   <extends>QWidget</extends>
   <header>baseglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
